Extending/Property-Editors/Build-a-Block-Editor.md

```diff
@@ -98,17 +96,17 @@ In the following example the layout object "MyOwn.UnicornBlocksEditor" is of typ
 
 ### Basic knowledge for understanding how to work with Block Editor data
 
-To help your Block Editor manage the Data Structure presented above we have made a Model Object called BlockEditorModelObject which helps manage the basic parts of a Block Editor.
+To help your Block Editor manage the Data Structure presented above, we have made a Model Object called BlockEditorModelObject which helps manage the basic parts of a Block Editor.
 
 To get a better understanding of what the Model Object does for you, we need to look at some usages of the Model Object.
 
 ### Maintain and work with the Layout of a Block Editor
 
-The `layout` of a Block Editor can be of any structure. Therefor the Model Object(BlockEditorModelObject) cannot maintain this data. This makes our usage of Model Object a bit more complex as we offen will be giving the Model Object a reference to an entry of the `layout` for then to performe an action and if what went well we might have to reflect this change back to the `layout`.
+The `layout` of a Block Editor can be any structure. Therefore the Model Object (BlockEditorModelObject) cannot maintain this data. This makes our usage of Model Object a bit more complex as we often will be giving the Model Object a reference to an entry of the `layout` to then perform an action, which may then need to reflect changes back to the `layout`.
```

Comment:
Small suggestion, to shorten the last sentence just a bit.
Let me know what you think!

```suggestion
The `layout` of a Block Editor can be any structure. Therefore the Model Object (BlockEditorModelObject) cannot maintain this data. This makes our usage of Model Object a bit more complex as we often will be giving the Model Object a reference to an entry of the `layout` to then perform an action. This action may then need to reflect changes back to the `layout`.
```